Service/UserApi/UserService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\ClientHelperBundle\Service\UserApi;
+
+final class UserService extends UserApiService
+{
+    /**
+     * @param array<mixed> $context
+     * @return array<mixed>
+     */
+    public function getUsers(array $context): array
+    {
+        $client = $this->createClient(['X-Auth-Token' => $context['authToken']]);
+        $response = $client->get(\sprintf('/api/user-profiles'));
```

Comment:
```suggestion
        $response = $client->get('/api/user-profiles');
```

Reply:
prone to error, this dependency will never be detected to core, we should think  about a bigger solution. It is crucial that this url can be detected when refactoring